lib/tasks/sip_refer.js

```diff
@@ -94,7 +94,11 @@ class TaskSipRefer extends Task {
         }
         if (status >= 200) {
           this.referSpan.setAttributes({'refer.finalNotify': status});
-          await this.performAction({refer_status: 202, final_referred_call_status: status});
+          await this.performAction({refer_status: 202, final_referred_call_status: status})
+            .catch((err) => {
+              this.logger.error(err, 'TaskSipRefer:exec - error performing action finalNotify');
+              this.notifyTaskDone();
```

Comment:
It seems that `this.notifyTaskDone()` will be called twice in the case of an error, because it would have already been called from the line below.  So I think this can be removed

Reply:
ah yes we're not throwing a new error so it will continue through,